assets/ui.css

```diff
@@ -1,3 +1,14 @@
+.close-button {
```

Comment:
Maybe just to be safe?
```suggestion
.respec-modal .close-button {
```